Dialog/CreditEditor_TableView.cpp

```diff
@@ -0,0 +1,237 @@
+﻿#include "Dialog/CreditEditor_TableView.h"
+#include "Dialog/CreditsEditDialog.h"
+#include <QTableView>
+#include <QStyledItemDelegate>
+#include <QMenu>
+#include <iostream>
+#include <QPainter>
+#include <QKeyEvent>
+
+
+CreditEditor_TableView::CreditEditor_TableView(QWidget *parent) :
+    QTableView(parent)
+{
+    menu=new QMenu("Menu", this);
+    action_one_tile = new QAction("One Tile",menu);
+    action_upper_tile = new QAction("Upper Tile",menu);
+    action_lower_tile = new QAction("Lower Tile",menu);
+    action_delete = new QAction("Delete",menu);
+
+    menu->addAction(action_one_tile);
+    menu->addAction(action_upper_tile);
+    menu->addAction(action_lower_tile);
+    menu->addAction(action_delete);
+
+    this->setContextMenuPolicy(Qt::CustomContextMenu);
+    connect(this, SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(showContextMenu(const QPoint &)));
+    connect(menu, SIGNAL(triggered(QAction *)), this, SLOT(doAction(QAction *)));
+}
+
+
+
+/// <summary>
+/// Called each time a cell is changed
+/// </summary>
+/// <param name="topLeft">
+/// The top left QModelIndex
+/// <param name="bottomRight">
+/// Unused
```

Comment:
If these are unused, let's start the function with:
```
(void) bottomRight;
(void) roles;
```
This will reduce the number of warnings.